.travis.yml

```diff
@@ -22,8 +22,9 @@ install:
   - composer install --prefer-dist
 
 before_script:
-  - ~/.phpenv/versions/5.6/bin/php -S localhost:8000 -t $(pwd) > /dev/null 2> /tmp/webserver_output.txt &
-  - export REPO_URL=http://localhost:8000/
+  - "~/.phpenv/versions/5.6/bin/php -S localhost:8000 -t $(pwd) > /dev/null 2> /tmp/webserver_output.txt &"
```

Comment:
Why this and next line show as changed? Maybe you've changed indentation by mistake?